expression/builtin_info.go

```diff
@@ -383,8 +383,56 @@ type benchmarkFunctionClass struct {
 	baseFunctionClass
 }
 
+// getFunction See https://dev.mysql.com/doc/refman/5.7/en/information-functions.html#function_benchmark
 func (c *benchmarkFunctionClass) getFunction(ctx sessionctx.Context, args []Expression) (builtinFunc, error) {
-	return nil, errFunctionNotExists.GenWithStackByArgs("FUNCTION", "BENCHMARK")
+	if err := c.verifyArgs(args); err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	// Syntax: BENCHMARK(loop_count, expression)
+	// Now that eval result of input arg is useless, we can define by the same eval type of input arg here.
```

Comment:
It's not useless, we should use the child `EvalType` to construct the function to avoid a `CAST()`.

Reply:
Comment improved in [`90a4828`](https://github.com/pingcap/tidb/commit/90a4828261fe8f0764cc2e25f8df14555c193973#diff-d4d67a9a31a3ecdc4f783ae7b09dd2f8R392)